src/index.js

```diff
@@ -2,16 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
```

Comment:
- [ ] It would be better if our `index.js` file follows the following pattern:
  - [ ] Has 2 `<Link>s` in the navbar which leads to the booklist and categories locations. Also, note that the navbar might be in its own component
  - [ ] Has 2 `<Route>`s, one for the BookList component and the other for the Categories component

For example; here is how it will look
```
<Navbar>
  <Link1>link 1</Link1>
  <Link2>link 2</LInk2>
</Navbar>
<Routes>
  <Route1 />
  <Route2 />
</Routes>
```

Reply:
please check the code i already added